RELEASING.md

```diff
@@ -0,0 +1,79 @@
+# Releasing Guidelines for Cucumber Godog
+
+This document provides guidelines for releasing new versions of Cucumber Godog. Follow these steps to ensure a smooth and consistent release process.
+
+## Versioning
+
+Cucumber Godog follows [Semantic Versioning]. Version numbers are in the format `MAJOR.MINOR.PATCH`.
+
+### Current (for v0.MINOR.PATCH)
+
+- **MINOR**: Incompatible API changes.
+- **PATCH**: Backward-compatible new features and bug fixes.
+
+### After v1.X.X release
+
+- **MAJOR**: Incompatible API changes.
+- **MINOR**: Backward-compatible new features.
+- **PATCH**: Backward-compatible bug fixes.
+
+## Release Process
+
+1. **Update Changelog:**
+    - Open `CHANGELOG.md` and add an entry for the upcoming release formatting according to the principles of [Keep A CHANGELOG].
+    - Include details about new features, enhancements, and bug fixes.
+
+2. **Run Tests:**
+    - Run the test suite to ensure all existing features are working as expected.
+
+3. **Manual Testing for Backwards Compatibility:**
+    - Manually test the new release with external libraries that depend on Cucumber Godog.
+    - Look for any potential backwards compatibility issues, especially with widely-used libraries.
+    - Address any identified issues before proceeding.
+
+4. **Tagging:**
+    - Create a new annotated tag for the release.
+      ```
+      git tag -a vMAJOR.MINOR.PATCH -m "Version MAJOR.MINOR.PATCH"
+      ```
+
+5. **Push Changes:**
+    - Push the new tag to the GitHub repository.
+      ```
+      git push origin vMAJOR.MINOR.PATCH
+      ```
+
+6. **Create Release on GitHub:**
+    - Go to the [Releases] page on GitHub.
+    - Click on "Draft a new release."
+    - Tag version should be set to the newly created tag.
+    - Title the release using the version number (e.g., "vMAJOR.MINOR.PATCH").
+    - Click 'Generate release notes'
```

Comment:
I'm not sure if these 3 steps should b simplified or not, but all them can be done with just step 6 (create release and tag a version at once).

Reply:
yes, let's keep it simple, updated